changelogs/current.yaml

```diff
@@ -82,6 +82,11 @@ minor_behavior_changes:
 
 bug_fixes:
 # *Changes expected to improve the state of the world and are unlikely to have negative effects*
+- area: config
```

Comment:
I think this needs to be runtime guarded because functionally there's a different behavior on the data plane with a currently valid config.

Reply:
Sounds good, added a runtime guard named `http_route_connect_proxy_by_default`. Please let me know if you would prefer a different name.